config.json

```diff
@@ -59,6 +59,12 @@
       "label": "Custom Logger Example",
       "path": "custom-logger-example",
```

Comment:
same here

Reply:
What is wrong with custom-logger sample?